src/views/components/set-user-info/set-user-info.js

```diff
@@ -71,56 +71,78 @@ export class SetUserInfo extends Component {
     const { isOpen:isOpenState } = this.state;
 
     const isOpen = isOpenProps && isOpenState;
+    const { i18n } = this.props;
```

Comment:
good job on removing new `I18n` instance, but i'd rather import it from the main instance `../I18n.js` files
instead of drilling it as prop.